scripts/prebuild.sh

```diff
@@ -4,36 +4,10 @@ set -e
 # debug log
 set -x
 
-cd core
-flutter pub get && dart run build_runner build --delete-conflicting-outputs
+find . -name "pubspec.yaml" -exec bash -c '
+  (cd $(dirname "$0") && flutter pub get && dart run build_runner build -d)
+' {} \;
```

Comment:
please keep the order of execute build-runner

Reply:
Can't, only way to do this is hardcoding

Reply:
<img width="817" alt="Screenshot 2024-10-09 at 10 01 45" src="https://github.com/user-attachments/assets/5c27d653-aa02-427f-aa68-73c30d65be71">

IMO, we should specify the modules then run the loop.

Reply:
> <img alt="Screenshot 2024-10-09 at 10 01 45" width="817" src="https://private-user-images.githubusercontent.com/80730648/374786953-5c27d653-aa02-427f-aa68-73c30d65be71.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0NDUxMjgsIm5iZiI6MTcyODQ0NDgyOCwicGF0aCI6Ii84MDczMDY0OC8zNzQ3ODY5NTMtNWMyN2Q2NTMtYWEwMi00MjdmLWFhNjgtNzNjMzBkNjViZTcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDAzMzM0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3NjkxMGEyNGUxMjM0NTU0ZDYwMjJmNTMxMzRlNDRmZjQxMmFiNDQxY2ExNTIxN2JlZGE1MWVhMTI1ODJlMTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jSZ8Z89sUogmDi289qP0ov34dgzzNjN9u8X3hPfo84E">
> 
> IMO, we should specify the modules then run the loop.

Yes, this is my original plan